### PR TITLE
Fixed right-to-left languages action view animation

### LINKED ISSUE #2879: Right to left languages panel animation and css problems
### Expected behavior
The action panel should work the same way left to right as it does right to left. 

### Actual Behavior
The action panel is missing borders and animation isn't reversed properly. When the panel animates out it should animate left to the right for right to left languages. See video, it should make sense. 

panel animation

cart_completed_and_new_issue_ _reactioncommerce_reaction

### Steps to Reproduce the Behavior
Open the panel for orders or account/user admin
Watch the panel animate in incorrectly. 
Also, notice the borders are in the wrong places

### Versions

Node: 4.5.0
NPM: 5.4.1
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.12.0
Reaction: 1.5.0
Reaction branch: marketplace

### PR DESCRIPTION
**Pull request by aanchirinah**
Resolves #2879,

So initially the animation for the action view panel was transitioning from right to left for all languages especially right-to-left languages. I added a class method named `handleLeftToRightAnimation` that will set the animation state to enable the action view animation to transition from left to right for right-to-left languages. The method is invoked in `componentWillReceiveProps` which will be triggered whenever the end-user changes the language in the navbar.

#### How to Test
- Sign in as admin or a user
- Set language on the navbar to Arabic or Hebrew
- Open the panel for orders or any other menu item on the side panel.
- Watch as the animation transitions from left to right.

### CODE REVIEW

**Comment by aanchirinah** [comment]
@mikemurray , I need your assistance with how to get the css for the borders of the action view panel to appear properly when the language is switched to a right-to-left option. The anomaly can be seen below from a screenshot @rymorgan posted in the issue comments, 
cart_completed_and_new_issue_ _reactioncommerce_reaction

**Comment by mikemurray** [comment]
@aanchirinah

use `console.less` use `html.rtl` in front of some other selector for when its `rtl`

ex:

```
html.rtl .rui.admin.action-view {
  // and then do your border tweaks.
} 
```

File: imports/plugins/core/dashboard/client/components/actionView.js [hunk 1]
```diff
@@ -189,6 +205,27 @@ class ActionView extends Component {
     }
   }
 
+  componentWillReceiveProps(nextProps) {
+    // Array containing abbreviations of right-to-left languages
+    const rightToLeftLanguages = ["ar", "he"];
```
Comment (zenweasel) [inline imports/plugins/core/dashboard/client/components/actionView.js]: This info should not be hardcoded into a single component, it should be moved to a common file somehow

File: imports/plugins/core/dashboard/client/containers/actionViewContainer.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import { Meteor } from "meteor/meteor";
```
Comment (zenweasel) [inline imports/plugins/core/dashboard/client/containers/actionViewContainer.js]: Import order

**Comment by zenweasel** [comment]
When log in I immediately get this error:

```

meteor.js?hash=6d285d84547b3dad9717a7c89c664b61b45ea3d8:942 TypeError: Cannot read property 'lang' of undefined
    at composer (actionViewContainer.js:29)
    at tracker.js:19
    at Tracker.Computation._compute (tracker.js?hash=997515fa2d5b0530ba07741da556c4b36963ef3b:339)
    at new Tracker.Computation (tracker.js?hash=997515fa2d5b0530ba07741da556c4b36963ef3b:229)
    at Object.Tracker.autorun (tracker.js?hash=997515fa2d5b0530ba07741da556c4b36963ef3b:613)
    at tracker.js:17
    at Object.Tracker.nonreactive (tracker.js?hash=997515fa2d5b0530ba07741da556c4b36963ef3b:640)
    at tracker.js:16
    at Container._subscribe (compose.js:85)
    at Container.componentWillMount (compose.js:27)
```

**Comment by aanchirinah** [comment]
@zenweasel , I have tested this branch out thoroughly can you please test again, I have fixed the requested changes too.

**Comment by zenweasel** [comment]
I have tried this again and am still getting the same error:

```
[brenthoover@sloat: 09:16:49 ~/Projects/js/reaction  (audax-fix-issue-2879 $) - [2062]
 »» git pull
Already up-to-date.
[brenthoover@sloat:
```

**Comment by aanchirinah** [comment]
@zenweasel , I have redone my implementation using a querySelector to determine when RTL styles are implemented.

**Pull request merged**